tests/client/amo/containers/TestCategoryPage.js

```diff
@@ -0,0 +1,57 @@
+import deepcopy from 'deepcopy';
+
+import { mapStateToProps } from 'amo/containers/CategoryPage';
+
+
+describe('CategoryPage.mapStateToProps()', () => {
+  const state = {
```

Comment:
I think it would be less fragile to create this state by dispatching real actions. It should be just as easy to do. Example: https://github.com/mozilla/addons-frontend/issues/1371